test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateLights.py

```diff
@@ -42,6 +42,9 @@ def setUpClass(cls):
             'VP2RenderDelegateLightsTest', 'baseline')
 
         cls._testDir = os.path.abspath('.')
+        
+        mayaVer = int(cmds.about(majorVersion=True))
+        cls._suffix = '-2024' if mayaVer <= 2024 else ''
```

Comment:
@seando-adsk The version is set to use files with -2024 suffix for Maya 2024 and older.